tests/integration/conftest.py

```diff
@@ -1,18 +1,32 @@
+import discord.ext.test
 import pytest
 
 from poglink.bot import ConfigurableBot
 
 
 @pytest.fixture()
-def sample_bot(bans_url_1, rates_url_1):
+def sample_bot(bans_url_1, rates_url_1, event_loop):
+    # Define intents required to publish embeds
+    intents = discord.Intents.default()
+    intents.members = True
+
     bot = ConfigurableBot(
         ".",
         {
             "bans_url": bans_url_1,
             "rates_urls": [rates_url_1],
             "data_dir": "tests/data",
         },
+        loop=event_loop,  # Ensure pytest uses same event loop for async functions as bot
+        intents=intents,
```

Comment:
Had to add a bit of extra configuration to the sample bot in order for it to work with `pytest`